### ISSUE TITLE
platformio?

### ISSUE THREAD

**Issue by drahdiwaberl** [DeuxVis/Lora-TTNMapper-T-Beam#8]
i'm aware that this isn't really an issue of your code, but maybe you could help - having more experience with this board..

i'm trying to get this built with platformio and actually seem to be getting quite close it, with the following ini:

```
[env:ttgo-t-beam]
platform = espressif32
board = ttgo-t-beam
framework = arduino
lib_deps = 852, 870, 1655
```

but on compiling i get:
error: 'class WiFiClass' has no member named 'mode'

unless uncommenting:
WiFi.mode(WIFI_OFF);

researching a little it seems the WIFI_OFF mode seems to be special to the ESP version of the wifi library http://esp8266.github.io/Arduino/versions/2.1.0-rc1/doc/libraries.html#wifi-esp8266wifi-library so it seems the wrong one is picked. any idea why and how to correct this? or maybe it's something more basic anyways?

tia,markus.

**Comment by DeuxVis** [comment]
No, sorry can't help you on that, I have no experience with Platformio and no idea how it works.

**Comment by DeuxVis** [comment]
I'm keeping that issue open to remind me to give platformio a try one day.

In the mean time if you find a solution to your problem please let us know.

**Comment by drahdiwaberl** [comment]
currently debugging a different issue with an ESP8266 board i stumbled upon this
https://community.platformio.org/t/esp32-wifi-library-dependency/2906
which sounds quite similar.

(thought i add it as kind of note to self, currently can't try if it's a solution that might be of help here as well..)
